solr/test-framework/src/java/org/apache/solr/util/SolrJettyTestRule.java

```diff
@@ -52,13 +53,18 @@ protected void after() {
       try {
         jetty.stop();
       } catch (Exception e) {
-        throw new RuntimeException(e);
+        try {
+          throw new IOException(e);
+        } catch (IOException ex) {
+          throw new RuntimeException(ex);
+        }
       }
       jetty = null;
     }
   }
 
-  public void reset() throws Exception {
+  @Deprecated // Prefer not to have this.
```

Comment:
How would we remove these?    Is that part of the scope of this PR or it's own PR?

Reply:
It not part of the scope of this PR.

Reply:
It communicates we have a method that we don't really love; debatable.  If we had the time to tend to each and every subclass of SolrJettyTestBase, I think we wouldn't have needed this.  A comment would be helpful Joshua.